content/02.main-text.md

```diff
@@ -42,12 +42,27 @@ The official version of the manuscript is _forked_ by individual contributors.
 A contributor then adds and revises files, grouping these changes into _commits_.
 When the changes are ready to be reviewed, the series of commits are submitted as a _pull request_ through GitHub, which notifies other authors of the pending changes.
 GitHub's review interface allows anyone to comment on the changes, globally or at specific lines, asking questions or requesting modifications as depicted in @url:https://medium.com/towards-data-science/opportunities-and-obstacles-for-deep-learning-in-biology-and-medicine-6ec914fe18c2.
-Conversations during review can reference other pull requests, issues, or authors, linking the relevant people and content.
+Conversations during review can reference other pull requests, issues, or authors, linking the relevant people and content, as illustrated in Figure @fig:workflow.
 Reviewing batches of revisions that focus on a single theme is more efficient than independently discussing isolated comments and edits, and it helps maintain consistent content and tone across different authors and reviewers.
 Once all requested modifications are made, the manuscript maintainers, a subset of authors with elevated GitHub permissions, formally approve the pull request and merge the changes into the official version.
-`TODO: need a figure with a flowchart showing this process`
 The process of writing and revising material can be orchestrated through GitHub with a web browser.
 
+![
+**Deep Review editing workflow.**
+Any reader can become a contributor by proposing a change through a pull request.
+In this example, the contributor opens an issue to discuss a manuscript modification.
+A maintainer and additional participant provide feedback, and the maintainer recommends creating a pull request to update the text.
+The contributor creates the pull request, it is reviewed by a maintainer and a participant, and the contributor updates the pull request in response.
+Once the pull request is approved, the maintainer merges the changes into the official version of the manuscript.
+](images/workflow.svg){#fig:workflow width="100%"}
```

Comment:
Let's reduce the width, since currently the text in the figure is much bigger than other text in the PDF. I wonder if we should do `width="3.5in"` or whatever inch width is most appropriate to make the font size 11 or 12.

Reply:
75% width looked pretty good in one of my local builds so I changed to that in ea62ce6.  Everything else was approved so I'll merge once this build passes.

I suggest we keep #8 open until we finish discussing the GitHub review figure.

Reply:
I prefer absolute width to relative width. The absolute width should be set to the width that the figure is the right size. I think absolute has some nice advantages when scaling the page IIRC.